src/bitcoin-cli.cpp

```diff
@@ -259,6 +259,14 @@ static void http_error_cb(enum evhttp_request_error err, void *ctx)
     reply->error = err;
 }
 
+static int8_t NetworkStringToId(const std::string& str)
+{
+    for (size_t i = 0; i < NETWORKS.size(); ++i) {
```

Comment:
we're returning an `int8_t`, we might as well iterate over one
```suggestion
    for (int8_t i{0}; i < NETWORKS.size(); ++i) {
```

Reply:
Not sure but I think this would cause a conversion from size_t to int8_t on each iteration, rather than once for `return i`. Doesn't seem worth it.

Reply:
the problem is likely that the method return `int8_t` for an index - unnecessary, should likely be `size_t` or `int` - not important